lms/djangoapps/courseware/views/views.py

```diff
@@ -634,17 +643,18 @@ def course_about(request, course_id):
         course = get_course_with_access(request.user, permission, course_key)
         course_details = CourseDetails.populate(course)
         modes = CourseMode.modes_for_course_dict(course_key)
+        home_page_url_name = course_home_url_name(request)
```

Comment:
I feel like this should use the same name (`course_home_url_name`) or a shortened name, but not a different name.